ui/src/features/WorkspaceTriggers/hooks.ts

```diff
@@ -20,27 +21,29 @@ export default () => {
   const [triggerToBeDeleted, setTriggerToBeDeleted] = useState<
     Trigger | undefined
   >(undefined);
-  const { useGetTriggersInfinite, useDeleteTrigger } = useTrigger();
-
+  const { useGetTriggers, useDeleteTrigger } = useTrigger();
+  const TRIGGERS_FETCH_RATE_PER_PAGE = 15;
```

Comment:
This is already defined in the useAPI or useQuery. If its needed here either export it from there or add it to the `global-constants.ts` file and import appropriately.